script/Arcueid/Day 5/05_00_ARC05_3.txt

```diff
@@ -46,12 +47,12 @@ And besides, how are we going to find him in the first place?"
 [sha:c6778861bba95a892d3abcb32df6ebcf07de85c1]{
 -- Page 220, Offset 9141.
 -- 一番確実な戦法は、わたしの体力が戻りきるまで逃げること。
-The safest strategy until my strength returns to normal is to run.
+For now, the safest strategy would be to stay on the run, at least until I regain some of my strength.
 }
 [sha:dd144ea0616cb8ec28646e761dfe79416e1fe996]{
 -- Page 220, Offset 9142.
 -- あと三日もすればわたしの規模は三割ぐらいまで回復する。それだけの出力があれば、わたしひとりでもヴローヴを倒しきれる」
-After three days, my <strength|L i f e  S c a l e    > will be restored to about 30% of its original level. With that much power, I'll be able to defeat Vlov on my own."
+After three days, my <strength|L i f e  S c a l e    > will be back to around 30% of its original level. That should be enough for me to defeat Vlov on my own."
```

Comment:
I think we can drop the ruby and just have Lifescale here now that it's been beaten into the reader during d4